mongo/integration/unified/operation.go

```diff
@@ -206,6 +206,22 @@ func (op *operation) run(ctx context.Context, loopDone <-chan struct{}) (*operat
 	// Unsupported operations
 	case "count", "listIndexNames", "modifyCollection":
 		return nil, newSkipTestError(fmt.Sprintf("the %q operation is not supported", op.Name))
+	case "createKey":
```

Comment:
Can we put these above the `Unsupported operations` with a new `CSFLE operations` comment?